conf/livy.conf

```diff
@@ -75,7 +75,13 @@
 # livy.server.recovery.state-store.url =
 
 # If Livy can't find the yarn app within this time, consider it lost.
-# livy.server.yarn.app-lookup-timeout = 30s
+# livy.server.yarn.app-lookup-timeout = 60s
+# When the cluster is busy, we may fail to launch yarn app in 30s, then it would cause session
```

Comment:
+# When the cluster is busy, we may fail to launch yarn app in app-lookup-timeout, then it would cause session